account_product_fiscal_classification/__manifest__.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2014-Today GRAP (http://www.grap.coop)
+# Copyright (C) 2016-Today La Louve (<http://www.lalouve.net/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Product - Fiscal Classification",
+    "summary": "Simplify taxes management for products",
+    "version": "16.0.1.0.1",
+    "category": "Accounting",
+    "author": "Akretion,GRAP,La Louve,Odoo Community Association (OCA)",
+    "maintainers": ["legalsylvain"],
+    "website": "https://github.com/OCA/account-fiscal-rule",
+    "license": "AGPL-3",
+    "depends": ["account", "account_usability"],
```

Comment:
Is it required to depends in `account_usability`? I understand `account_usability `but I don't use this module and I would like to maintain without using it.

Just a question. Thank you! 😄

Reply:
Hi @rafaelbn. account_usability module (previously named account_menu module in v<16) is a simple module that adds menu entries. It is required to put the menu of the classification template in the correct place. IMO : 

- the module is well maintained. (present in all the version)
- it is easy (mostly xml)

Reply:
> Hi @rafaelbn. account_usability module (previously named account_menu module in v<16) is a simple module that adds menu entries. It is required to put the menu of the classification template in the correct place. IMO : 
> 
> - the module is well maintained. (present in all the version)
> - it is easy (mostly xml)

Hello, Sylvain, I have the same doubt for other modules I want to port to v16. One of the doubts I have is if people will complain if it doesn't play well with Odoo EE. Do you know if account_menu works decently along with Odoo EE?

Reply:
Hi. 

> Do you know if account_menu works decently along with Odoo EE?

I don't see why there would be any bugs.

Reply:
yes, I understand. That being said lot's of OCA modules got little modifications to play it nice with Odoo EE as many OCA users also have customers with EE. As for me I don't have access to a v16 EE repo and that's not a big deal/blocker for me, just "a better have" in case you knew the answer about compatibility.